### DISCUSSION TITLE
Unable to post the blog on my website through a self hosted ghost page

### DISCUSSION THREAD

**Discussion by ikarus-akhil** [docker-library/ghost#404] [General]
I have created a ghost docker image using the `ghost:5` image as a base image, pushed it to ECR, and then deployed it using ECS. The image has been deployed to ECR since November 30 and I recently just updated my nginx image (which behaves as a reverse proxy) and then I ran the task definition again, but since then I have been unable to post my recent blog as it is freezing as soon as I click the publish button.
I am getting the following console error:
![Console Error](https://github.com/docker-library/ghost/assets/113896034/a632844c-6600-433d-97da-0dc8836dcde3)

So then what I did was I checked the console logs of the docker image and got the following error:
![CLI Error](https://github.com/docker-library/ghost/assets/113896034/c5f2951a-4365-41db-af8c-5bfef82a69ab)

**Comment by tianon** [comment]
That's weird -- is the `newsletters` behavior that appears to be erroring here built-in, or some kind of optional plugin?

(Missing a key seems like some database migration failed to run, but I'm 100% guessing.)

**Reply by ikarus-akhil** [reply]
I have not done any migration so it cannot be a migration error.

**Comment by ikarus-akhil** [comment]
So I was able to solve this issue by just making a new image of my application, which updated the ghost version in the image and the problem was resolved.
